src/js/core.js

```diff
@@ -395,6 +395,9 @@ function MediumEditor(elements, options) {
         AnchorPreview: AnchorPreview
     };
 
+    MediumEditor.Util = Util;
+    MediumEditor.Selection = Selection;
```

Comment:
If there was a way of denotaing "private" or "protected-but-exposed" methods and namespaces, we could simply mark this as such and not have to worry too much about it. I don't really like that it is `MediumEditor.Util` (would prefer MediumEditor.util and .selection respectively) ... but went with 1:1 initial name matching.  So yes, we're exposing this hunk of functionality that was previously there but obscured ... so I'm of the thinking that by exposing them to the world as-is we open them up to scrutiny. I think people will forgive semVer violations for internal tooling that was only exposed to make it so they could even use it ... also there is the deprecated method in there, which means you can deprecate anything and remove cruft at major bumps, keeping around the "old broken" versions until that happens.